modules/webbrowser/properties/propertycefsynchronizer.cpp

```diff
@@ -33,6 +33,13 @@
 #include <modules/webbrowser/properties/ordinalpropertywidgetcef.h>
 #include <modules/webbrowser/properties/stringpropertywidgetcef.h>
 
+#include <modules/webbrowser/webbrowsermodule.h>
+
+#include <warn/push>
+#include <warn/ignore/all>
+#include "include/cef_parser.h"
```

Comment:
generally "include" would not be in the header path. Do you have the correct include paths setup?

Reply:
Use same as CEF examples, so yes :)